### ISSUE TITLE
Assertion failed on parsing malformed Eiffel file (main/entry.c:1108)

### ISSUE THREAD

**Issue by kcwu** [universal-ctags/ctags#543]
How to reproduce
```
$ echo -e 'class feature d alias"\t' > min.e
$ ctags min.e
ctags: main/entry.c:1108: void initTagEntryFull(tagEntryInfo *const, const char *const, unsigned long, const char *, fpos_t, const char *): Assertion `name == NULL || strchr (name, '\t') == NULL' failed.
ctags: main/entry.c:1108: parsing min.e:1 as Eiffel
Aborted
```

this is found by afl-fuzz

**Comment by masatake** [comment]
Thank you for all the reports. I will look them later.

**Comment by masatake** [comment]
Closed by #582.
